rectools/visuals/metrics_app.py

```diff
@@ -0,0 +1,128 @@
+import typing as tp
+
+import ipywidgets as widgets
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+from IPython.display import display
+
+from rectools import Columns
+
+
+class MetricsApp:
+    """
+    Jupyter widgets app for metric visualization and comparison.
+    Do not create instances of this class directly. Use `MetricsApp.construct` method instead.
+    """
+
+    def __init__(
+        self,
+        df_metrics_data: pd.DataFrame,
+        chart_width: tp.Union[float, int] = 800,
+        chart_height: tp.Union[float, int] = 600,
+        show_legend: bool = False,
```

Comment:
let's change to `True` here and in `construct`

Reply:
Done